### DISCUSSION TITLE
How to get back to the last position after Goto?

### DISCUSSION THREAD

**Discussion by Zhou-Yicheng** [helix-editor/helix#2113] [Q&A] [answered]
I am using helix and lsp to view in a project. But I'm confused with file, buffer or something. If I just jump within a file using **goto**, how do I **go back**?

**Comment by sudormrfbin** [comment, marked as answer]
Positions you jump to are stored in the jumplist. To go back to a previous location use `ctrl-o`, to go forward use `ctrl-i`.

**Reply by Zhou-Yicheng** [reply]
Thank you!
